File: NbaTracker.Api/NbaTracker.Common/DataContracts/BoxScore.cs
namespace NbaTracker.Common.DataContracts;

public class BoxScore
{
    public string GameId { get; set; }
    public DateTime GameTime { get; set; }
    public Arena Arena { get; set; }
    public Team HomeTeam { get; set; }
    public Team AwayTeam { get; set; }
    public ICollection<PlayerBoxScore> HomeTeamPlayers { get; set; }
    public ICollection<PlayerBoxScore> AwayTeamPlayers { get; set; }
}

File: nba-tracker-app/src/app/DataContracts/BoxScore.ts
import { Team } from './Team';
import { PlayerBoxScore } from './PlayerBoxScore';

export interface BoxScore {
  gameId: string;
  gameTime: Date;
  homeTeam: Team;
  awayTeam: Team;
  homeTeamPlayers: PlayerBoxScore[];
  awayTeamPlayers: PlayerBoxScore[];
}
